GOVERNANCE.md

```diff
@@ -0,0 +1,163 @@
+
+In extreme cases, maintainers can lose their status by a vote of the maintainers per the voting process below.
+
+## Decision Making Process
+
+Decisions about the future of Amundsen are made through discussion with all community members, from the newest user to the most experienced maintainer. All non-sensitive project management discussion takes place on the project issue tracker system. Occasionally, sensitive discussion occurs on a private channel of our Slack.
+
+To ensure that the project is not bogged down by endless discussion and continual voting, the project operates a policy of lazy consensus. This allows the majority of decisions to be made without resorting to a formal vote.
+
+### Lazy consensus
+
+Decision making typically involves the following steps:
+
+- Proposal
+- Discussion
+- Vote (if consensus is not reached through discussion)
+- Decision
+
+Any community member can make a proposal for consideration by the community. To initiate a discussion about a new idea, they should create an issue or submit a PR implementing the idea to the issue tracker. This will prompt a review and, if necessary, a discussion of the idea. The goal of this review and discussion is to gain approval for the contribution. Since most people in the project community have a shared vision, there is often little discussion to reach consensus.
+
+In general, as long as nobody explicitly opposes a proposal or PR, it is recognized as having the support of the community. This is called lazy consensus - that is, those who have not stated their opinion explicitly have implicitly agreed to the proposal's implementation.
+
+Lazy consensus is a fundamental concept within the project. This process allows a large group of people to reach consensus efficiently as someone with no objections to a proposal need not spend time stating their position.
+
+For lazy consensus to be effective, it is necessary to allow at least 48 hours before assuming that there are no objections to the proposal. This requirement ensures that everyone is given enough time to read, digest, and respond to the proposal. This time period is chosen to be as inclusive as possible of all participants, regardless of their location and time commitments.
+
+### Voting
+
+Not all decisions can be made using lazy consensus. Issues such as those affecting the strategic direction or legal standing of the project must gain explicit approval in the form of a vote. Every member of the community is encouraged to express their opinions in all discussions and all votes. However, only project maintainers have binding votes for the purposes of decision making.
+
```

Comment:
One clarification: Have you all thought about where this poll would be conducted? Would it be slack or over email list? If it is going to Slack, have we thought of how to make sure it doesn't get lost in other conversations, esp considering we have people from different parts of world (different timezones) active on slack.

Reply:
We think that for large architectural changes and features, the future RFC repository and the PRs open there will be the right discussion forum.

For the voting of maintainers, we'll do it on a private channel in the Slack, sharing the results afterwards on the public channels